checker/src/org/checkerframework/checker/experimental/regex_qual/RegexQualifiedTransfer.java

```diff
@@ -82,7 +83,8 @@ public RegexQualifiedTransfer(QualAnalysis<Regex> analysis) {
                         thenStore.insertValue(firstParam, new RegexVal(0));
                     }
                 } catch (FlowExpressionParseException e) {
-                    assert false;
+                    ErrorReporter.errorAbort("RegexQualifiedTransfer.visitMethodInvocation: could not parse " +
```

Comment:
This exact same line is duplicated several times throughout these changes. Can you think of a way to reduce this duplication, e.g. by using a method in FlowExpressionParseException?